idunn/api/places.py

```diff
@@ -0,0 +1,66 @@
+from apistar.exceptions import NotFound
+from elasticsearch import Elasticsearch
+
+from idunn.api.poi import POI
+from idunn.api.pois import fetch_es_poi
+from idunn.api.place import Place, Admin, Street, Address
+from idunn.utils.settings import Settings
+
+def get_index(type, settings):
+    return {
+        "admin": settings['PLACE_ADMIN_INDEX'],
+        "street": settings['PLACE_STREET_INDEX'],
+        "address": settings['PLACE_ADDRESS_INDEX'],
+        "poi": settings['PLACE_POI_INDEX'],
+    }.get(type, "_all")
```

Comment:
you can put this dict as a global variable.

And:
* we should never you `_all` but `munin` if we want all the indexes. else if the ES cluster is used for other thing we'll get strange results.
* I'm not a fan of getting all the indexes if the type is not known. if the index is `bob` I want an error, (I think it's on `type=None` that we want all indexes).

Reply:
ok, done